djangocms_snippet/migrations/0011_cms4_plugin_data_migration.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 2.2.24 on 2021-08-31 10:45
+import logging
+from django.db import migrations
+
+
+logger = logging.getLogger(__name__)
+
+
+def cms4_migration(apps, schema_editor):
+    plugin_count = 0
+    SnippetPtr = apps.get_model("djangocms_snippet", "SnippetPtr")
+
+    for snippet_plugin in SnippetPtr.objects.all():
+        snippet = snippet_plugin.snippet
+        grouper = snippet.snippet_grouper
+        snippet_plugin.new_snippet = grouper
+        snippet_plugin.save()
+        plugin_count += 1
+    logger.info(f"Migration completed, created {plugin_count} Snippet plugins")
```

Comment:
Same as above, I don't think that we need to keep the logs in for the final version.